src/app/parts-editor/components/dialog-database/dialog-database.component.html

```diff
@@ -0,0 +1,20 @@
+<h1 i18n>Parts missing from database</h1>
```

Comment:
Naming conventions of dialog can be better. This name is ambiguous. It doesn't tell dialogs purpose.  Its purpose is to add missing parts, so it would be much better to call it  MissingPartsDialog, PartListDialog, DbPartListDialog